### DISCUSSION TITLE
Submit your certification

### DISCUSSION THREAD

**Discussion by extinctsion** [github/india#349] [Students]
After completing a module successfully, [Azure Fundamentals (AZ-900)] it says to submit your certification. So the question is what to submit in the email as it is prompted right after clicking on the link?

![image](https://user-images.githubusercontent.com/67048929/136614123-841f79f1-e178-473a-99d0-f6f35cbdcfeb.png)

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
